From a2e05a70ab1a5dfa1118814f061b9154fd62282a Mon Sep 17 00:00:00 2001
From: Ashwin H <Ashwin.H@amd.com>
Date: Mon, 26 Jun 2023 15:15:32 +0000
Subject: [PATCH 09/14] drivers/spi/spidev.c: Add pensando,cpld device tree
 compat entry

The Pensando ASICs uses a spidev interface to access the board CPLD.
This commit adds support for the entry in spidev.c
---
 drivers/spi/spidev.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/spi/spidev.c b/drivers/spi/spidev.c
index 859910ec8..6d36402a9 100644
--- a/drivers/spi/spidev.c
+++ b/drivers/spi/spidev.c
@@ -682,6 +682,7 @@ static const struct of_device_id spidev_dt_ids[] = {
 	{ .compatible = "lwn,bk4" },
 	{ .compatible = "dh,dhcom-board" },
 	{ .compatible = "menlo,m53cpld" },
+	{ .compatible = "pensando,cpld" },
 	{},
 };
 MODULE_DEVICE_TABLE(of, spidev_dt_ids);
-- 
2.17.1

